src/helpers/surveyHelpers.tsx

```diff
@@ -0,0 +1,13 @@
+export const combineLanguages = (
+  languages: string[],
+  localeObj?: Record<string, string | undefined>
+) => {
+  if (!localeObj) {
+    return;
+  }
+
+  return languages
+    .map((language) => localeObj?.[language])
+    .filter((langTitle) => langTitle?.length)
+    .join(' / ');
```

Comment:
instead of joining with / the result should be a new line if there is polish text and italic style for polish.
i do not know if this is handled in a follow up already or if it is even possible in an easy way with our existing structures

Reply:
it is done partially.

for the `SectionHeader`s it is still with the ` / ` to avoid two `SectionHeader`s following each other.
for the overview it is still with a ` /  ` as well.

I was allowed to do layout changes in the call about the wireframe, in case some things do not fit properly.

That is also the reason for the dates being in two lines, instead of left and right (future PR)

Reply:
(also SVA2-19 does not speak of any wireframe yet, as this ticket was pre wireframe)

Reply:
> That is also the reason for the dates being in two lines, instead of left and right (future PR)

i assumed already that this will be necessary when seen the wireframe first time